things/model/src/main/java/org/eclipse/ditto/things/model/devops/events/WotValidationConfigCreated.java

```diff
@@ -0,0 +1,178 @@
+/*
+ * Copyright (c) 2025 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.ditto.things.model.devops.events;
+
+import java.time.Instant;
+import java.util.Optional;
+import java.util.function.Predicate;
+
+import javax.annotation.Nullable;
+import javax.annotation.concurrent.Immutable;
+
+import org.eclipse.ditto.base.model.entity.metadata.Metadata;
+import org.eclipse.ditto.base.model.headers.DittoHeaders;
+import org.eclipse.ditto.base.model.json.FieldType;
+import org.eclipse.ditto.base.model.json.JsonParsableEvent;
+import org.eclipse.ditto.base.model.json.JsonSchemaVersion;
+import org.eclipse.ditto.base.model.signals.events.Event;
+import org.eclipse.ditto.base.model.signals.events.EventsourcedEvent;
+import org.eclipse.ditto.json.JsonFactory;
+import org.eclipse.ditto.json.JsonField;
+import org.eclipse.ditto.json.JsonFieldDefinition;
+import org.eclipse.ditto.json.JsonObject;
+import org.eclipse.ditto.json.JsonObjectBuilder;
+import org.eclipse.ditto.json.JsonParseException;
+import org.eclipse.ditto.json.JsonPointer;
+import org.eclipse.ditto.json.JsonValue;
+import org.eclipse.ditto.things.model.devops.ImmutableWotValidationConfig;
+import org.eclipse.ditto.things.model.devops.WotValidationConfigId;
+
+/**
+ * Event that is emitted when a WoT validation config is created.
+ */
+@Immutable
+@JsonParsableEvent(name = WotValidationConfigCreated.NAME, typePrefix = WotValidationConfigEvent.TYPE_PREFIX)
+public final class WotValidationConfigCreated extends AbstractWotValidationConfigEvent<WotValidationConfigCreated>
+        implements WotValidationConfigModifiedEvent<WotValidationConfigCreated> {
+
+    public static final String NAME = "wotValidationConfigCreated";
+    public static final String TYPE_PREFIX = WotValidationConfigEvent.TYPE_PREFIX;
```

Comment:
This line should not be needed - `TYPE_PREFIX` is extended and automatically "in scope" from the parent class.